src/features/auth/forgottenPassword/ResetPasswordExpiredLink/ResetPasswordExpiredLink.tsx

```diff
@@ -35,10 +37,17 @@ export function ResetPasswordExpiredLink(props: Props) {
     }
   }
 
+  const renderResendEmailButton = () => (
+    <ButtonPrimaryWhite
+      title={t`Renvoyer l'email`}
+      onPress={() => resetPasswordEmailQuery()}
```

Comment:
```suggestion
      onPress={resetPasswordEmailQuery}
```

le mieux est d'éviter de passer une nouvelle fonction quand c'est possible.